courseme/util/json.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+"""JSON utility functions"""
+
+from __future__ import absolute_import
+import json as j
+
+from datetime import datetime
+
+class _CustomEncoder(j.JSONEncoder):
+    """Provide serialisation for types not handled by default encoder"""
+
+    def default(self, obj):
+        if isinstance(obj, datetime):
+            encoded_object = obj.isoformat()
+        else:
+            encoded_object = j.JSONEncoder.default(self, obj)
+        return encoded_object
+
+
+def loads(s, *args, **kwargs):
```

Comment:
The `*args, **kwargs` is an example where I've let an implementation detail leak into the interface.  ie - I'm allow the caller to pass in additional arguments, and I'm passing those on verbatim to the standard library `json` module.  In this case, it's a judgement call, as I know that other `json` module implementations all mimic the standard library's interface very closely.